### DISCUSSION TITLE
why wallet node and full node need to sync respectively ?

### DISCUSSION THREAD

**Discussion by Firesuiry** [Chia-Network/chia-blockchain#8579] [Support]
it is very slow when i create a new wallet key.

**Comment by loppefaaret** [comment]
it is just the nature of our software, that the node and wallet keeps seperated databases... 
when you say it is very slow creating a new key - are you creating many new keys ? it is rarely needed to creating many keys ? and if you are referring to the fact that it takes a while to sync the new key, i would only assume so

**Reply by Firesuiry** [reply]
yes， your assuming is right. I want to create a website supporting chia to pay. and I want to create a addition key to test.
Now I have sync the two key and all is ok.
thank you for your reply. 
and now i have a new question about the transcation record. 
https://github.com/Chia-Network/chia-blockchain/issues/8612
can you help me to solve this problem?

**Comment by BrandtH22** [comment]
@Firesuiry , the newest version of chia has fixes in place to resolve this issue including a lite wallet option for running the client. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
